AdventOfCode2016/Day1.fs

```diff
@@ -0,0 +1,84 @@
+﻿module Day1
+
+open System
+open System.IO
+
+type Turn = Left | Right
+
+type Direction = North | East | West | South
+
+type Instruction = {
+    Direction: Turn;
+    Distance: int;
+}
+
+type Coordinate = {
+    X : int;
+    Y : int;
+}
+
+type State = {
+    Position: Coordinate;
+    Facing: Direction;
+}
+
+let parseDirection (c:char) =

```

Comment:
Not an issue with the code at all - but I generally prefer to have my functions in a seperate module from my types. I generally will have them in a module with `[<RequiredQualifiedAccess>]` attribute applied. This forces modules to be used.

That then makes content look like this:
```
initialState
|> Seq.map Movement.parseDirection
```

or similar